### PR TITLE
Move events group name to component params (#61)

### PR DESCRIPTION
**Pull request by olegbaturin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #61

### CODE REVIEW

**Comment by olegbaturin** [comment]
For the first consideration

File: config/params-console.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'yiisoft/yii-event' => [
+        'eventsGroup' => 'events-console',
```
Comment (vjik) [inline config/params-console.php:7]:
```suggestion
        'eventsConfigGroup' => 'events-console',
```

File: config/params-web.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'yiisoft/yii-event' => [
+        'eventsGroup' => 'events-web',
```
Comment (vjik) [inline config/params-web.php:7]:
```suggestion
        'eventsConfigGroup' => 'events-web',
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## 2.0.1 under development
 
-- no changes in this release.
+- Enh #61: Move events group name to component params (@olegbaturin)
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- Enh #61: Move events group name to configuration parameters (@olegbaturin)
```

**Review by vjik — CHANGES_REQUESTED** [review]

**Comment by olegbaturin** [comment]
What's wrong with rector?

**Review by vjik — APPROVED** [review]

**Comment by vjik** [comment]
> What's wrong with rector?

Rector workflow don't work in PR from forks. So far we have not able to solve this problem. May be has idea how fix it?

**Comment by vjik** [comment]
Thanks 👍

**Comment by olegbaturin** [comment]
> > What's wrong with rector?
> 
> Rector workflow don't work in PR from forks. So far we have not able to solve this problem. May be has idea how fix it?

maybe helps
https://github.com/actions/checkout/issues/455

**Pull request merged**